daemon/execdriver/native/create.go

```diff
@@ -212,6 +212,20 @@ func (d *driver) setupRlimits(container *configs.Config, c *execdriver.Command)
 }
 
 func (d *driver) setupMounts(container *configs.Config, c *execdriver.Command) error {
+	userMounts := make(map[string]bool)
```

Comment:
IMO, Go should have templates so users could create whatever datatypes they want ;)